web/api/queries/query.c

```diff
@@ -537,8 +537,6 @@ static inline void do_dimension_fixedstep(
         , time_t before_wanted
         , uint32_t options
 ){
-    RRDSET *st = r->st;
```

Comment:
Hi Stelio, should this one be inside an `ifdef NETDATA_INTERNAL_CHECKS` ? It is used later in the code (line 580 now, 582 before).

Reply:
yes will fix it...